### ISSUE TITLE
Investigate NCBI FTP upload issue

### ISSUE THREAD

**Issue by tudorgroza** [EBISPOT/goci#219]

**Comment by tudorgroza** [comment]
@ljwh2 @jdhayhurst - I had a look at the logs and then in the code and there's unfortunately not much to dig into. It seems that we just can't upload files to `ftp-private.ncbi.nlm.nih.gov` - but I can't tell you why because the code does not log the reason. It goes through 7 attempts and then abandons it altogether. We would probably have to get in touch with the US folks to understand why is this failing.

**Comment by ljwh2** [comment]
@JalMacArthur can you help with this? I don't know where this data is going

**Comment by JalMacArthur** [comment]
I agree that we will need to get in touch with NCBI to find out why this is failing.

**Comment by ljwh2** [comment]
@JalMacArthur what happens to the data when it gets to NCBI, is this going to PheGenI?

**Comment by JalMacArthur** [comment]
Yes, but it may go into other resources too.

**Comment by ljwh2** [comment]
I have forwarded relevant emails to gwas-dev

**Comment by ljwh2** [comment]
@sprintell need to find out why failure emails are still arriving
